src/app/portlets/dot-porlet-detail/dot-contentlets/dot-contentlets.component.spec.ts

```diff
@@ -59,26 +62,37 @@ describe('DotContentletsComponent', () => {
         dotRouterService = de.injector.get(DotRouterService);
         dotIframeService = de.injector.get(DotIframeService);
         dotContentletEditorService = de.injector.get(DotContentletEditorService);
+        dotCustomEventHandlerService = de.injector.get(DotCustomEventHandlerService);
 
         spyOn(dotIframeService, 'reloadData');
         fixture.detectChanges();
     });
 
-    it('should call contentlet modal', async(() => {
-        const params = {
-            data: {
-                inode: '5cd3b647-e465-4a6d-a78b-e834a7a7331a'
-            }
-        };
-        setTimeout(() => {
-            expect(dotContentletEditorService.edit).toHaveBeenCalledWith(params);
-        }, 0);
-    }));
+    it(
+        'should call contentlet modal',
+        async(() => {
+            const params = {
+                data: {
+                    inode: '5cd3b647-e465-4a6d-a78b-e834a7a7331a'
+                }
+            };
+            setTimeout(() => {
+                expect(dotContentletEditorService.edit).toHaveBeenCalledWith(params);
```

Comment:
Is this being called on load?

Reply:
ngAfterViewInit https://github.com/dotCMS/core-web/blob/master/src/app/portlets/dot-porlet-detail/dot-contentlets/dot-contentlets.component.ts#L20